src/elisp/treemacs-mouse-interface.el

```diff
@@ -244,17 +244,17 @@ and ignore any prefix argument."
 (defun treemacs--tag-noselect (btn)
   "Return list of tag source buffer and position for BTN for future display."
   (cl-flet ((xref-definition
-             (identifier)
-             "Return the first definition of string IDENTIFIER."
-             (car (xref-backend-definitions (xref-find-backend) identifier)))
+              (identifier)
```

Comment:
Indentation shouldn't change here. The latest commit adds a .dir-locals file that should preclude unwanted diffs like this.